### DISCUSSION TITLE
Refund

### DISCUSSION THREAD

**Discussion by Pedro-Baracat** [aclap-dev/video-downloadhelper#1253] [License & Purchase requests]
### Your question

I downloaded the Google Chrome plug-in, but Google Chrome cannot be used. I want a refund.

Email: pedroo.baracat@gmail.com

License ID: 8dd142bde6c44753a2cc29773e3ae39b

### Addon & CoApp version and details

I downloaded the Google Chrome plug-in, but Google Chrome cannot be used. I want a refund.

Email: pedroo.baracat@gmail.com

License ID: 8dd142bde6c44753a2cc29773e3ae39b

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
